src/roseingrave/_sheets.py

```diff
@@ -70,6 +71,7 @@ def _error(msg):
         'credentials_filename': str(filepath),
         'authorized_user_filename': str(auth_user_path),
         'flow': gspread.auth.console_flow,
+        'client': gspreed.client.BackoffClient,
```

Comment:
This should be `'client_factory': gspread.client.BackoffClient`. Note the typo in `gspread`.

Reply:
Good catch, sorry about the sloppiness! 😅 
https://docs.gspread.org/en/latest/api/top-level.html#gspread.oauth